adcircpy/fort15.py

```diff
@@ -606,10 +606,10 @@ def get_tidal_forcing(self) -> str:
                         # f'{len(self.mesh.open_boundaries)}',
                     ]
                 )
-            for index, row in self.mesh.boundaries.ocean.gdf.iterrows():
                 for constituent in self.mesh.forcings.tides.get_active_constituents():
                     f.append(fort15_line(constituent))
-                    vertices = self.mesh.get_xy(crs='EPSG:4326').iloc[row.indexes, :].values
+                for index, row in self.mesh.boundaries.ocean.gdf.iterrows():
+                    vertices = self.mesh.get_xy(crs='EPSG:4326').loc[row.indexes, :].values
```

Comment:
Why was `iloc` changed to `loc`? Are the position of *xy* rows the same as the dataframe index?

Reply:
That is accidental. I copied lines from 1.2.2. Thanks!